index.js

```diff
@@ -116,6 +123,7 @@ app.get('/unsecure/users/:id', user.findById)
 app.get('/unsecure/users/:id/favorites', favorites.findFavoriteListsByUser)
 
 app.get('/unsecure/users/:userId/restaurants/visits', visits.allUserVisits)
+app.get('/unsecure/users/:userId/restaurants/:restaurantId/visits', visits.findByRestaurantId)
```

Comment:
Am a little tickled by this path as it could clash with the one just underneath.
Is the use case really to get per user per restaurant? Could it be simply per restaurant?

If not, I guess it is fine.

Reply:
It was to follow the design of the API. As the visits were tied to the user, I added the path the same way.

But I am totally fine with moving it under the restaurants endpoints.

Reply:
In fact, maybe should we have both. I'll check it out later.